### PR TITLE
2.3oss

### PR DESCRIPTION
**Pull request by kelseiv**
closes https://github.com/influxdata/docs-v2/issues/4134, https://github.com/influxdata/docs-v2/issues/4133, https://github.com/influxdata/docs-v2/issues/4141

### CODE REVIEW

File: content/influxdb/v2.3/reference/release-notes/influxdb.md [hunk 1]
```diff
@@ -0,0 +1,1816 @@
...
+- Vectorize logical operations in `map()`.
+- Enable `movingAverage()` and `cumulativeSum()` optimizations by default.
+
+##### Other highlights
+
+- Add `preview()` to experimental package for limiting return rows and tables (as opposed to just rows with `limit()`).
+- Add `date.scale()` to let users dynamically scale durations in dates.
+- Add [OpenTracing](https://opentracing.io/docs/overview/spans/) spans to Flux transformations. Lets you monitor Flux scripts more precisely.
+- Add `trace` option to Flux CLI.
+- Move `addDuration()` and `subDuration()` out of experimental to date package.
```
Comment (kelseiv) [inline content/influxdb/v2.3/reference/release-notes/influxdb.md:39]: @sanderson @sunbryely-influxdata looks like we need to move these 2 functions in docs

File: content/influxdb/v2.3/reference/release-notes/influxdb.md [hunk 2]
```diff
@@ -0,0 +1,1816 @@
...
+- Add location support to `date.truncate()`.
+- Add `_status` tag to PagerDuty record.
+- Refactor operator profile to be in the query statistics.
+
+### Bug fixes
+
+- Fix race condition when remote replicating deadlocks the remote writer that prevented writes to the target.
+- Resolve error when creating v1 auth with a nonexistent bucket ID.
```
Comment (kelseiv) [inline content/influxdb/v2.3/reference/release-notes/influxdb.md:47]: @samhld should we clarify authentication or authorization here?

File: content/influxdb/v2.2/security/disable-devel.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+description: >
+  Disable development features that may not be desirable in production.
+weight: 105
+menu:
+  influxdb_2_2:
+    parent: Security & authorization
+influxdb/v2.2/tags: [security, development]
+---
+
+By default, InfluxDB {{< current-version >}} enables functionality useful in developer environments. Depending on your site requirements, you may want to disable this functionality when running InfluxDB in production. To disable, use the following options:
```
Comment (samhld) [inline content/influxdb/v2.2/security/disable-devel.md:13]:
```suggestion
By default, InfluxDB {{< current-version >}} enables useful functionality that expose some level of information about your instance. Two of these are endpoints for observability of the health and activity of your instance. The third is the bundled UI. Depending on your site requirements, you may want to disable one or more of these when running InfluxDB in production. To disable, use the following configuration options:
```

**Review by samhld — APPROVED** [review]

File: content/influxdb/v2.3/reference/release-notes/influxdb.md (hunk 1 above)
Comment (sunbryely-influxdata) [reply content/influxdb/v2.3/reference/release-notes/influxdb.md:39]: @kelseiv just checked on this and we have already moved the `addDuration` `and subDuration` into the dates package. they were renamed from `addDuration` to `add` upon promotion to the date package

File: content/influxdb/v2.2/security/disable-devel.md [hunk 4]
```diff
@@ -10,7 +10,7 @@ menu:
 influxdb/v2.2/tags: [security, development]
 ---
 
-By default, InfluxDB {{< current-version >}} enables functionality useful in developer environments. Depending on your site requirements, you may want to disable this functionality when running InfluxDB in production. To disable, use the following options:
+By default, InfluxDB {{< current-version >}} enables useful functionality that exposes some level of information about your instance. Two of these are endpoints for observability of the health and activity of your instance. The third is the bundled UI. Depending on your site requirements, you may want to disable one or more of these when running InfluxDB in production. To disable, use the following configuration options:
```
Comment (jdstrand) [inline content/influxdb/v2.2/security/disable-devel.md:13]: These changes LGTM, thanks!

File: content/influxdb/v2.3/reference/release-notes/influxdb.md (hunk 1 above)
Comment (kelseiv) [reply content/influxdb/v2.3/reference/release-notes/influxdb.md:39]: Great news, thanks @sunbryely-influxdata --I'll clarify in the release notes

**Pull request merged**